### ISSUE TITLE
A Discriminative Feature Learning Approach for Deep Face Recognition

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#411]
## 一言でいうと

画像分類において、単純に分類確率(softmax)だけでなく、当該クラスの中心からの距離をlossに組み込もうという提案(中心は最初に計算してしまうのでなく、学習中ミニバッチにより随時更新されていく)。

### 論文リンク

http://ydwen.github.io/papers/WenECCV16.pdf

### 著者/所属機関

Yandong Wen, Kaipeng Zhang, Zhifeng Li, Yu Qiao
* Shenzhen Key Lab of Comp. Vis. & Pat
* Shenzhen Institutes of Advanced Technology
* The Chinese University of Hong Kong

## 手法

中心からの距離

![image](https://user-images.githubusercontent.com/544269/30411449-efb74a86-994b-11e7-9369-53a05ce3df3a.png)

中心の更新

![image](https://user-images.githubusercontent.com/544269/30411459-ffd022d0-994b-11e7-8d58-ce073633f6ea.png)

softmaxのlossにλの割合で組み込む

![image](https://user-images.githubusercontent.com/544269/30411465-05fd9728-994c-11e7-9f99-7a6d580e035d.png)

## 結果

![image](https://user-images.githubusercontent.com/544269/30411482-1d1f101c-994c-11e7-9b4d-e66b5744531f.png)

実験に使っているネットワーク自体はかなり単純で、lossの効果が効いていると推察される。

![image](https://user-images.githubusercontent.com/544269/30411544-7f690bec-994c-11e7-8541-1faf4e85872b.png)

## コメント

元論文は顔の分類だが、こちらのブログで様々なデータセットで検証している。
https://medium.com/mlreview/experiments-with-a-new-loss-term-added-to-the-standard-cross-entropy-85b080c42446
